internal/config/config.go

```diff
@@ -73,8 +75,19 @@ type Config struct {
 	LogLevel slog.Level `yaml:"log"`
 }
 
+var reEnv = regexp.MustCompile(`\$\{[a-zA-Z_][a-zA-Z_0-9]*\}`)
```

Comment:
I know that it is not exported, but I would add a doc comment.

```suggestion
// reEnv is used to replace embedded environment variables.
var reEnv = regexp.MustCompile(`\$\{[a-zA-Z_][a-zA-Z_0-9]*\}`)
```

Reply:
done